docs/configs/site.js

```diff
@@ -50,14 +50,24 @@ export default {
           },
           {
             key: 'docs1',
-            text: '1.3.9',
-            link: '/en-us/docs/1.3.9/user_doc/quick-start.html',
+            text: '2.0.5',
+            link: '/en-us/docs/2.0.5/user_doc/guide/quick-start.html',
           },
           {
             key: 'docsHistory',
             text: 'Older Versions',
             link: '/en-us/docs/release/history-versions.html',
           },
+          {
+            key: 'pythonAPI',
+            text: 'PyDolphinScheduler',
+            link: '/python/index.html',
+          },
+          {
+            key: 'docsdev',
+            text: 'dev',
+            link: '/en-us/docs/dev/user_doc/about/introduction.html',
+          },
```

Comment:
```suggestion
```

Reply:
we should delete them in Chinese version config

Reply:
ok， I will delete it in Chinese documentation